liquibase-standard/src/test/java/liquibase/changelog/OfflineChangeLogHistoryServiceTest.java

```diff
@@ -28,16 +29,9 @@ public class OfflineChangeLogHistoryServiceTest {
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-    private File getResourceAsFile(String resourceName) {
```

Comment:
I was curious about the removal of `getResourceAsFile` and went through the git history. It looks like this method was never used (ever!). Thanks for removing it!